src/components/Review/ReviewSectionRowAction.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Grid } from 'semantic-ui-react'
+import { useRouter } from '../../utils/hooks/useRouter'
+import { ReviewSectionComponentProps } from '../../utils/types'
+import strings from '../../utils/constants'
+
+const ReviewSectionRowAction: React.FC<ReviewSectionComponentProps> = ({
+  action,
+  section: { details },
+  isAssignedToCurrentUser,
+  thisReview,
+}) => {
+  const {
+    location: { pathname },
+  } = useRouter()
+
+  const reviewPath = `${pathname}/${thisReview?.id}`
+  const reviewSectionLink = `${reviewPath}?activeSections=${details.code}`
+
+  const getContent = () => {
+    switch (action) {
+      case 'canContinue': {
+        if (isAssignedToCurrentUser) return <Link to={reviewSectionLink}>{strings.CONTINUE}</Link>
+
+        return <p>In Review</p>
+      }
+      case 'canView': {
+        if (isAssignedToCurrentUser) return <Link to={`${reviewSectionLink}`}>{strings.VIEW}</Link>
+        else return <Link to={`${reviewSectionLink}`}>{strings.VIEW}</Link>
+      }
+
+      case 'canStartReview': {
+        if (isAssignedToCurrentUser) return strings.START
```

Comment:
```suggestion
      case ReviewAction.canContinue: {
        if (isAssignedToCurrentUser) return <Link to={reviewSectionLink}>{strings.CONTINUE}</Link>

        return <p>In Review</p>
      }
      case ReviewAction.canView: {
        if (isAssignedToCurrentUser) return <Link to={`${reviewSectionLink}`}>{strings.VIEW}</Link>
        else return <Link to={`${reviewSectionLink}`}>{strings.VIEW}</Link>
      }

      case ReviewAction.canStartReview: {
        if (isAssignedToCurrentUser) return strings.START
```
Also need to import `ReviewAction` from types

Reply:
9800aa94306142062dcde84cc7d889d0a2e88b2d, i've made a seperate commit as it was easier to keep track of compilation error and do a fix all at once